src/control/xml/XmlPointNode.cpp

```diff
@@ -1,11 +1,11 @@
 #include "XmlPointNode.h"
 
-XmlPointNode::XmlPointNode(const char* tag) : XmlNode(tag)
+XmlPointNode::XmlPointNode(const char* tag)
+ : XmlNode(tag),
+   points(NULL),
+   timestamp(0)
```

Comment:
is it like setting timestamp=0 ? I've never seen this before

Reply:
Yes, this is the standard C++ way, I didn't even know that setting in the header file does work at all, seems to be a new C++11 feature (or a G++ Extension)